### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by skothari-tibco**

### CODE REVIEW

File: activity/kafka/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: '2'
```
Comment (fm-tibco) [inline activity/kafka/docker-compose.yml]: If this is just for testing, we should put it in a subdirectory... any ideas?  'travis' maybe? or ".travis"?  thoughts?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#  List all the components in activity, trigger and function
```
Comment (fm-tibco) [inline test.sh]: If this is just for testing, we should put it in a subdirectory... any ideas?  'travis' maybe? or ".travis"?  thoughts?

File: activity/jsexec/go.mod [hunk 3]
```diff
@@ -10,4 +10,4 @@ require (
 	golang.org/x/text v0.3.2 // indirect
 )
```
Comment (fm-tibco) [inline activity/jsexec/go.mod]: Remove this file from PR

File: activity/jsexec/examples/microgateway/api/go.sum [hunk 4]
```diff
@@ -96,4 +96,4 @@ gorgonia.org/dawson v1.0.0/go.mod h1:Px1mcziba8YUBIDsbzGwbKJ11uIblv/zkln4jNrZ9Ws
 gorgonia.org/gorgonia v0.9.0-beta/go.mod h1:qucT7YHm/2OuSHWEw/6Je/LQ5htRJNQJ1+qpB58fY8c=
```
Comment (fm-tibco) [inline activity/jsexec/examples/microgateway/api/go.sum]: Remove this file from PR

File: activity/jsexec/go.sum [hunk 5]
```diff
@@ -29,4 +29,4 @@ golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGm
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
```
Comment (fm-tibco) [inline activity/jsexec/go.sum]: Remove this file from PR

File: activity/kafka/go.sum [hunk 6]
```diff
@@ -50,4 +50,4 @@ golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
```
Comment (fm-tibco) [inline activity/kafka/go.sum]: Remove this file from PR

File: activity/rest/go.sum [hunk 7]
```diff
@@ -21,4 +21,4 @@ go.uber.org/zap v1.9.1/go.mod h1:vwi/ZaCAaUcBkycHslxD9B2zi4UTXhF60s6SWpuDF0Q=
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405 h1:yhCVgyC4o1eVCa2tZl7eS0r+SDo693bJlVdllGtEeKM=
 gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
```
Comment (fm-tibco) [inline activity/rest/go.sum]: Remove this file from PR

File: trigger/rest/go.mod [hunk 8]
```diff
@@ -2,8 +2,7 @@ module github.com/project-flogo/contrib/trigger/rest
 
 require (
 	github.com/julienschmidt/httprouter v1.3.0
```
Comment (fm-tibco) [inline trigger/rest/go.mod]: Remove this file from PR, it shouldn’t have a dependency on 0.10.1

File: trigger/rest/go.sum [hunk 9]
```diff
@@ -1,13 +1,24 @@
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```
Comment (fm-tibco) [inline trigger/rest/go.sum]: Remove this file from PR

File: trigger/rest/go.mod (hunk 8 above)
Comment (skothari-tibco) [reply trigger/rest/go.mod]: I updated it as having 0.9.4 was breaking test.

File: activity/jsexec/examples/microgateway/api/go.sum (hunk 4 above)
Comment (skothari-tibco) [reply activity/jsexec/examples/microgateway/api/go.sum]: I tried  :( ; this won't go even though its the same.

File: activity/jsexec/go.mod (hunk 3 above)
Comment (skothari-tibco) [reply activity/jsexec/go.mod]: I tried  ; this won't go even though its the same.

File: activity/jsexec/go.sum (hunk 5 above)
Comment (skothari-tibco) [reply activity/jsexec/go.sum]: Same thing

**Pull request merged**